### DISCUSSION TITLE
Is there a way to test if a variable is set?

### DISCUSSION THREAD

**Discussion by bilogic** [deployphp/deployer#3383] [Help needed]
Hi,

2 related questions:

1. How do I test if `a` is set like this `host(...)->set("a", "value")`? `get("a")` gives me null
2. Can I write `{{ a ?? b }}`, i.e. if a is null then use b instead?

Thank you.

**Comment by antonmedv** [comment]
Nice idea. Will add it. Right now there is a has() func. https://deployer.org/docs/7.x/api#has
